features/notices/python/notice.py

```diff
@@ -38,7 +38,7 @@ def main():
     # You must assign `Register` to a variable (even if you don't run
     # `del` on it later) or the callback goes out of scope and does nothing.
     #
-    updated = Tf.Notice.Register(Usd.Notice.ObjectsChanged, update, stage)
+    updated: Tf.Notice.Listener = Tf.Notice.Register(Usd.Notice.ObjectsChanged, update, stage)
```

Comment:
Technically speaking, the use of annotations on variable assignments like this isn't necessary for mypy to work correctly, because the `types-usd` stubs provide the output type of the method.  I've included them in some places because they act as a kind of assertion:  if the output type of the method as defined by the stubs disagrees with the annotated type, then mypy will generate an error.